modules/planning/tasks/deciders/lane_change_decider/lane_change_decider.h

```diff
@@ -65,7 +65,8 @@ class LaneChangeDecider : public Decider {
       const ReferenceLineInfo* const reference_line_info);
 
  private:
-  common::Status Process(Frame* frame) override;
+  common::Status Process(Frame* frame,
```

Comment:
If not modifying, better use const reference than pointer;
If modifying, better use * const.

Reply:
Agree on the const reference part. But IMO it's OK to use `*` instead of `* const`, as redirecting a pointer is virtually never allowed. So we just save several bytes :)

Reply:
Agree. Better use const ReferenceLineInfo& reference_line_info here.